fantasy_football_project/context_processors.py

```diff
@@ -0,0 +1,4 @@
+from waffle import flag_is_active
+
+def entry_lock(request):
+    return {'entry_lock': flag_is_active(request, 'entry_lock')}
```

Comment:
Don't think this is needed. Try https://waffle.readthedocs.io/en/stable/usage/templates.html#flags